app/lib/actions/pulp3/orchestration/repository/delete.rb

```diff
@@ -0,0 +1,17 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module Repository
+        class Delete < Pulp3::Abstract
+          def plan(repository, smart_proxy)
+            sequence do
+              plan_action(Actions::Pulp3::Repository::DeleteRemote, repository, smart_proxy)
```

Comment:
We should probably delete the distributors here too

Reply:
as an orchestrated action i mean

Reply:
Yeah, I was wondering how to ... orchestrate... that - wasn't sure if this was the right place or not.